scripts/build.sh

```diff
@@ -5,10 +5,10 @@ set -e
 
 # build
 if [ "$SL_BUILD" = true ]; then
-  yarn workspace snapshot-listener build
+  pnpm --filter ./packages/snapshot-listener build
 elif [ "$DL_BUILD" = true ]; then
-  yarn workspace discord-bot build
+  pnpm --filter ./packages/discord-bot build
 else
-  NODE_OPTIONS=--max_old_space_size=4096 yarn workspace commonwealth bundle
-  yarn workspace commonwealth build
+  NODE_OPTIONS=--max_old_space_size=4096 pnpm --filter ./packages/commonwealth bundle
+  pnpm --filter ./packages/commonwealth build
```

Comment:
this is still building incrementally and will need project references - see `build` script in `/packages/commonwealth` using the --build option